stanford_courses.info.yml

```diff
@@ -0,0 +1,55 @@
+name: 'Stanford Courses'
+type: module
+description: 'Courses content type'
+core_version_requirement: ^8.8 || ^9
+version: 8.2.7
+package: Stanford
+dependencies:
+  - datetime
```

Comment:
These should all be namespaced with their packages. For example: https://github.com/SU-SWS/jumpstart_ui/blob/8.x-1.x/jumpstart_ui.info.yml#L7

Reply:
namespaced and alphabetized for clarity